.github/workflows/haskell-ci.yml

```diff
@@ -38,6 +38,11 @@ on:
         required: false
         # For an explanation of `--ghc-options='-j'`, see Note [Parallelism].
         default: --ghc-options='-j' --enable-tests
+      fetch-depth:
+        description: Number of commits to fetch. Passed to `actions/checkout`.
```

Comment:
It might be worth making a note that a value of `0` fetches _all_ commits (which is a bit counterintuitive).

Reply:
I should say more about *why* I'm hesitant to do so: I think there's just no clear line delineating how much we should document these things, so I feel that it's easier to do none at all. This particular quirk is documented upstream (arguably, not clearly enough), so it seems fine to forward interested readers there.